### ISSUE TITLE
Data Errors in Connector

### ISSUE THREAD

**Issue by LlanosPastor** [hellowearemito/dataconnector-facebook#21]
Hello!! First of all, thank you very much for the connector :)

I have an issue with some errors in the data exported to datastudio. Impressions, click and cost do work fine. Reach, frecuency, ctr and cpc don't.

This is the data I'm connecting to:

![image](https://user-images.githubusercontent.com/42376022/44087381-fac9d60c-9fbf-11e8-8ee6-088776253f08.png)

And this are the results the connector returns to datastudio:

![image](https://user-images.githubusercontent.com/42376022/44087471-3d727f54-9fc0-11e8-9ff3-4778f6e6a0c9.png)

Any idea why this may happen?

Thanks in advance!! 👍

**Comment by djozsef** [comment]
Hello @LlanosPastor,

I believe your problem derives from the way data studio interprets our source. See #20

**Comment by LlanosPastor** [comment]
not really, that thread says errors in the sums. Of course I do not expect the total to be the same as the data in the facebook panel, the errors are in campaigns one per one.

**Comment by barcaplasilova** [comment]
Hi @djozsef , thank you very much for the connector. Great Job! But unfortunatelly, I have the same problem as a @LlanosPastor. Reach, frecuency, ctr and cpc doesnt match with facebook data. I am afraid, this is not just a problem with sum. Have you find any solution? 
Thanks in advance :-)

**Comment by LlanosPastor** [comment]
@barcaplasilova the problem is: the connector counts the individual ads information and counts each ad individually, and since 1 user can view ads from different ad groups, facebook counts this user as 1, the connector counts 2 users if 1 user sees 2 different ads. Hope that makes it clear :)
